### DISCUSSION TITLE
Multi-Zone with custom domains?

### DISCUSSION THREAD

**Discussion by lardissone** [vercel/next.js#15024] [Help]
I need to create these three frontends:

- **admin**: https://admin.domain.com/
- **tenants**: https://tenantdomain.com/admin
- **students**: https://tenantdomain.com/

I'm planning to use **tenants** and **students** on the same deploy using multi-zones (pretty similar as the multi zones [example](https://github.com/vercel/next.js/blob/canary/examples/with-zones)).
I'm not sure if I could add the **admin** to the same deployment so that I can reuse components.

Is there a way to say "for this specific build use this alias" so that I can run the **admin** in its own domain?

*Update*: I know you can do that with HTTP proxies, but not sure if it's possible with Vercel.

**Comment by softmarshmallow** [comment]
Mine saying `The domain bridged.xyz is already used by project` - trying to make
- bridged.xyz
- bridged.xyz/docs

within as monorepo

**Reply by JClackett** [reply]
Having the same issue, can't select the same domain

**Comment by amosbastian** [comment]
Anyone managed to solve this yet? @lardissone @JClackett

**Reply by dnlfrst** [reply]
@amosbastian @lardissone @JClackett this setup works _exactly_ as showcased in the [example](https://github.com/vercel/next.js/tree/canary/examples/with-zones). The project for `https://tenantdomain.com/admin` will be deployed first in a separate project on Vercel. Then, you copy the deployment URL of that project (i.e., something like `https://tenantdomain-admin.vercel.app`) and add it to the `.env` file of `https://tenantdomain.com/`'s project and push the change to git. Afterward, you create a second project on Vercel for `https://tenantdomain.com/`. You assign your custom domain (i.e., `tenantdomain.com`) to the project of `https://tenantdomain.com/` but _not_ to the project for `https://tenantdomain.com/admin`, because it will already be working automagically.
